DIPs/001/README.md

```diff
@@ -0,0 +1,38 @@
+# Minimum Viable Governance
+This document describes the Minimum Viable Governance of Social Dist0rtion Protocol.
+
+## Goverance
+Governance is the sum of all Dist0rtion Improvement Proposals (DIPs) contained in the master branch of this repository.
+
+### Improve it
+Anyone can open a pull request to this repository to submit a new DIP. If the pull request is approved by the majority of the owners, it is then merged to master, becoming part of the governance.
+
+## Funds
+The funds of the DAO are managed by the multi signature wallet deployed in the Main Ethereum Network at the address `0x527621278422FFf45A66f7086bAeAC770CF12b69`
+
+The multi signature wallet defines a *set of owners* and a *number of required confirmations* to execute transactions.
+
+## Proposals
+Anyone can make a **proposal** by opening an issue in any repository of the Social Dist0rtion Protocol organization on GitHub.
+
+The proposal must contain:
+- The *problem* being addressed.
+- The *solution* proposed.
+
+The proposal may contain:
+- An *estimated effort*.
+- The *payment conditions*.
+
+The author of the proposal has the responsibility to gauge interest around their work, but it is not an obligation of the DAO to take their work into consideration.
+
+If the proposal is to be resolved on-chain (payments, minting, adding and removing owners, ...), owners may vote "yes" by confirming the proposed transaction in the multi signature wallet.
+
+Any other proposal can be voted by the owners in the comment section of the issue. If the number of "yes" votes is equal to the *number of required confirmations*, then the proposal is implemented.
```

Comment:
how is the number of required confirmations defined? Still majority based (everyone / 2 + 1)?

Reply:
It's defined in the multi signature wallet. I can rephrase it to:
> If the number of "yes" votes is equal to or greater than the *number of required confirmations* for the multisig wallet, then the proposal is implemented.

The threshold to transfer money is encoded in the *number of required confirmations* of the mutlisig wallet. In my opinion voting for other concerns should be handled with the same logic as executing transactions.

Reply:
th
```suggestion
Any other proposal can be voted by the owners in the comment section of the issue. If the number of "yes" votes is equal to the *number of required confirmations* defined multi signature wallet, then the proposal is implemented.
```